snap/snapcraft.yaml

```diff
@@ -51,6 +51,5 @@ parts:
 
       git config --global --add safe.directory '*'
       version="$(git describe --always --tags --exclude='weekly' | sed -e 's/^v//;s/-/+git/;y/-/./')"
-      [ -n "$(echo $version | grep "+git")" ] && grade=devel || grade=stable
```

Comment:
why removing this? is it because we won't rely on tags for this snap?

Reply:
Yep. And if  we leave it as is, since this repo is not gonna get tagged, the snap will always be `devel`. The `devel` grade cannot be promoted to `candidate`/`stable`...